functions/_common/downloadFunctions.ts

```diff
@@ -68,10 +68,12 @@ export async function fetchZipForGrapher(
 }
 function assembleCsv(grapher: Grapher, searchParams: URLSearchParams): string {
     const useShortNames = searchParams.get("useColumnShortNames") === "true"
+    const fullTable = grapher.inputTable // TODO: should this be table?
```

Comment:
Should this be `table` instead? `inputTable` includes all data, `table` is filtered by the time range set by the author (note that it's the author's time range, not the user's).

Not sure what's intended here, but the client side uses `table`, so regardless of what we choose, server and client should probably use the same table.

Reply:
I guess you can argue both ways but since this is the best way to get the full data I'm inclined to give the user everything we have in this case - i.e. stay with inputTable for the time beeing.

Reply:
Ok, then I'll update the client code to use `inputTable` as well!